code/game/objects/objs.dm

```diff
@@ -40,6 +45,9 @@ var/global/list/reagents_to_log = list(FUEL, PLASMA, PACID, SACID, AMUTATIONTOXI
 
 	var/has_been_invisible_sprayed = FALSE
 
+	// Tells is_honorable() which special_roles to respect.
+	var/honorable = HONORABLE_BOMBERMAN | HONORABLE_HIGHLANDER
```

Comment:
Does this have to be an /obj level variable? It's not going to be used that much, and there will rarely (if ever) be a need to edit it, so imo it would fit better as a proc.